xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/DXVAEnumeratorHD.cpp

```diff
@@ -395,3 +395,51 @@ bool CEnumeratorHD::IsBT2020Supported()
 
   return true;
 }
+
+bool CEnumeratorHD::QuerySDRtoSDRSupport() const
+{
+  // Not initialized yet
+  if (!m_pEnumerator)
+    return false;
+
+  if (!m_pEnumerator1)
+  {
+    CLog::LogF(LOGWARNING,
+               "ID3D11VideoProcessorEnumerator1 not available - assuming SDR output is possible.");
+    return true;
```

Comment:
I didn't express myself clearly in the description - the black screen is because IsFormatConversionSupported returns false. If that check is ignored, there is automatic driver fallback to full range output so it's not a black screen.

This block executes for Windows 8 and I didn't see a good reason to change from the current behavior (ie give the benefit of the doubt). At that time only SDR existed so support could be assumed to be present (even if incomplete with some fallbacks). There were no complaints before, it shouldn't start now.

On the other hand, exiting early in the other functions (passthrough and BT2020) is fine because HDR support didn't exists at the time of Windows 8 and couldn't be described in the older API describing transfer/primaries for the blit.